components/patina_smbios/src/manager/core.rs

```diff
@@ -113,9 +121,63 @@ impl SmbiosManager {
             minor_version,
             entry_point_64: RefCell::new(None),
             table_64_address: RefCell::new(None),
+            // Pre-allocated buffers start as None, set up during component init
+            table_buffer_addr: RefCell::new(None),
+            table_buffer_max_size: 64 * 1024, // 64KB default max size
+            ep_buffer_addr: RefCell::new(None),
         })
     }
 
+    /// Allocate buffers for SMBIOS table publication
+    ///
+    /// Allocates buffers for the SMBIOS table and entry point upfront to avoid
+    /// repeated allocations. The buffers are reused for all table publications.
+    ///
+    /// Also adds the Type 127 End-of-Table marker to maintain the invariant that
+    /// it's always the last record.
+    ///
+    /// Must be called at TPL_APPLICATION or lower to allow memory allocation.
+    ///
+    /// # Errors
+    ///
+    /// Returns `SmbiosError::AllocationFailed` if buffer allocation fails.
+    /// Returns `SmbiosError::HandleExhausted` if unable to allocate handle for Type 127.
+    #[cfg_attr(coverage_nightly, coverage(off))]
```

Comment:
```suggestion
    #[coverage(off)]
```

We are just using this form. Please amend everywhere